### PR TITLE
Refactor file command tests

### PR DESCRIPTION
**Pull request by aexolate**

### CODE REVIEW

**Review by Weiennn — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/FileCommand.java [hunk 1]
```diff
@@ -8,19 +8,28 @@
...
     /**
      * Checks whether the specified path is a valid data file path. A valid data file path should
-     * contain a file name, and the file name should end with ".json".
-     * @param path Data file path.
+     * contain a file name, and the file name should end with ".json". Note that a valid file path
+     * does not imply the file exists or can be read/written to.
      */
-    protected boolean isValidDataFilePath(Path path) {
-        Path fileNamePath = path.getFileName();
+    boolean isValidDataFilePath() {
```
Comment (Goh-Li-Ting) [inline src/main/java/seedu/address/logic/commands/FileCommand.java:21]: Is this supposed to have access modifiers? (and the `getFilePath` method)

File: src/test/java/seedu/address/logic/commands/FileCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    private static final String SPACED_FILE_NAME = "my data.json";
+    private static final String SPACED_FOLDER_NAME = "hello world/my data.json";
+
+
+
+    /**
+     * Tests the isValidDataFilePath function of abstract class FileCommand.
+     */
+    @Test
+    void isValidDataFilePath_invalidPath_falseReturned() {
```
Comment (Goh-Li-Ting) [inline src/test/java/seedu/address/logic/commands/FileCommandTest.java:40]: Are these methods supposed to have the `public` access modifiers?

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Pull request merged**